repository-data/webfiles/src/main/resources/site/freemarker/common/macro/results.ftl

```diff
@@ -101,13 +101,13 @@
 
 <#macro indicator item>
     <div class="push-double--bottom" data-uipath="ps.search-results.result">
-        <h3 class="flush zeta" data-uipath="ps.search-results.result.type" style="font-weight:bold">Indicator</h3>
+        <h3 class="flush zeta" data-uipath="ps.search-results.result.type" style="font-weight:bold"><@fmt.message key="labels.indicator"/></h3>
         <p class="flush">
             <a href="<@hst.link hippobean=item.selfLinkBean/>" title="${item.title}" data-uipath="ps.search-results.result.title">
                 ${item.title}
             </a>
         </p>
-        <#if item.assuredStatus><p class="flush zeta" data-uipath="ps.search-results.result.assured-status">Independently Assured by IGB (conditional)</p></#if>
+        <#if item.assuredStatus><p class="flush zeta" data-uipath="ps.search-results.result.assured-status">Independently Assured by IGB</p></#if>
```

Comment:
Could make this a label as well if it keeps changing?

Reply:
good point - done